### ISSUE TITLE
@angular/core and @angular/platform-browser-dynamic not listed in peerDependancies

### ISSUE THREAD

**Issue by freshfunkee** [thymikee/jest-preset-angular#140]
Hi,

I've just discovered this library. Kudos for the great work. It has really simplified our Jest setup while we are migrating our angular projects.

I thought I'd bring this to your attention, as I experienced it while installing the package.

We have a small repo with some simple angular api services. It's not a full application. It has a relatively straightforward setup. When I had the jest-preset-angular library configured with jest, I received an error : 

```
Cannot find module '@angular/platform-browser-dynamic/testing' from 'setupJest.js'
```

Our repo doesn't use this module directly, so I would have expected it to have been raised as a missing peer dependancy upon installing the package. From looking at the `setupJest.js` file, it also appears that `@angular/core` is also imported. Of course, angular projects should be expected to have this file, but this module should list this as a peer dependancy regardless.

**Comment by thymikee** [comment]
Would you be interested in adding them as peer deps to this project? 🙂 cc @AhnpGit

**Comment by freshfunkee** [comment]
Fixed with PR 141 https://github.com/thymikee/jest-preset-angular/pull/141

**Comment by just-jeb** [comment]
Is the issue back? 
It's not listed in [peerDependencies](https://github.com/thymikee/jest-preset-angular/blob/master/package.json#L48) now, however it is required in the [setup file](https://github.com/thymikee/jest-preset-angular/blob/master/src/config/setup-jest.ts#L22).  
For me it results in the following error (working with Yarn): 
```sh
Cannot find module '@angular/platform-browser-dynamic/testing' from '../../../../node_modules/jest-preset-angular/build/config/setup-jest.js'
```

Am I missing something?

**Comment by ahnpnl** [comment]
`"@angular/platform-browser-dynamic"` was missing in `peerDependencies` indeed.

**Comment by just-jeb** [comment]
So should this issue be reopened then?

**Comment by ahnpnl** [comment]
fixed in **9.0.3**, just published

**Comment by just-jeb** [comment]
Cool, thanks!
